### ISSUE TITLE
Cannot switch to MS Basic after using a sound from MuseSounds

### ISSUE THREAD

**Issue by msviolafangirl** [musescore/MuseScore#20643]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Bug description

In the event that I want to switch from a sound in the MuseSounds pack to a sound in the MS basic soundfont, I cannot get it to happen. I am using a screen reader. I don't know if non screen reader users are affected.

### Steps to reproduce

1. Go to mixer. Choose an instrument that is currently associated with a MuseSounds instrument.
2. Change the sound to MS Basic.
3. Notice that I can't get MS basic checked, and when I press right arrow to open the MS basic submenu, it lands me in the instrument section I was last examinining. For example, after it says Ms basic, I press right arrow and get all the brass sounds, but it won't let me move up just one level to get to the instrument families. See screen recording for details.

### MuseScore Version

4.2

### Regression

I don't know

### Operating system

Windows 10

**Comment by MarcSabatella** [comment]
For me, if I navigate to Soundfonts, then press right arrow to open the submenu listing the soundfonts, then navigate to MS Basic, then press right arrow to open the submenu listing the families, then navigate to Brass, then press right arrow to open the list of sounds within the family, I can select any sound I want there.

**Comment by MarcSabatella** [comment]
Unfortunately it seems your screen recording didn't attach.

**Comment by msviolafangirl** [comment]
I just attached it. Sorry it took some time.

**Comment by MarcSabatella** [comment]
Thanks, but unfortunately this doesn't seems to show anything happening.  Perhaps because the mixer is scrolled down to the bottom and the sound selection menu is out of view, or perhaps because your screen recording software isn't capturing the menus at all (this can happen when capturing a window rather than full screen).

**Comment by msviolafangirl** [comment]
Yeah, that might be the issue. As a screen reader user, I cannot assess the position of windows or anything like that. I am using the Windows Xbox game bar to do the screen recording, and I'm pretty sure it's just doing window capture.

**Comment by DmitryArefiev** [comment]
@msviolafangirl I've tried this with 4.2.1 and Narrator on Windows11. Looks fine on my side. Navigation with right left arrows works for me. Here is screen recording:


https://github.com/musescore/MuseScore/assets/90187801/1e192887-970d-4121-a53f-a2840ad0a90f

**Comment by FrancRos31** [comment]
Am I the only one seeing that this report is closed, but no closing notifications appear in the history?

<img width="1154" height="188" alt="Image" src="https://github.com/user-attachments/assets/a771b8a7-676f-48a3-94b6-f762de031767" />

**Comment by cbjeukendrup** [comment]
Same for me... curious! Never had that before.

**Comment by jmwmusic** [comment]
There seems to be a glitch with msviolafangirl's account so she could not leave a comment.  It would be great if someone could look into this so she can assist with reporting accessiblity issues, currently she can't seem to comment or create issues. 

She wanted to leave a comment on this issue to say that it has been resolved for some time possibly around 4.3 and sorry she hadn't noticed to close it sooner.

**Comment by FrancRos31** [comment]
> There seems to be a glitch with msviolafangirl's account so she could not leave a comment. It would be great if someone could look into this so she can assist with reporting accessiblity issues, currently she can't seem to comment or create issues.

Has she tried deleting and reopening her account?

**Comment by cbjeukendrup** [comment]
Intuitively, I would not recommend that, because some sites keep usernames reserved forever even if the account has been deleted, making it impossible to create a new account with the same username. 

I've asked someone to check if this account got somehow blocked from our repository; still waiting for a conclusion about that. If it turns out not to be specific to our repository, then you will have to contact GitHub Support.

**Comment by FrancRos31** [comment]
That's definitely a major inconvenience! Just out of curiosity, @cbjeukendrup, are you able to reopen the issue, overriding the account problem with @msviolafangirl?

**Comment by cbjeukendrup** [comment]
It looks like I would be able to reopen this issue, but according to https://github.com/musescore/MuseScore/issues/20643#issuecomment-3146072746, that would not be necessary, right?

**Comment by FrancRos31** [comment]
Yeah, you're right, no need to reopen.
